src/config.books.json

```diff
@@ -21,7 +21,7 @@
     "defaultVersion": "5.1"
   },
   "b56bb9e9-5eb8-48ef-9939-88b1b12ce22f": {
-    "defaultVersion": "36.3"
+    "defaultVersion": "37.1"
```

Comment:
After entering https://archive.cnx.org/contents/b56bb9e9-5eb8-48ef-9939-88b1b12ce22f it shows `@38.3`